client/src/components/app.jsx

```diff
@@ -81,55 +111,41 @@ class App extends React.Component {
   }
 
   render () {
+    if (this.state.productId === null) {
+      return (
+        <div>
+          <div><strong>BAD REQUEST - INVALID URL</strong></div>
+          <br></br>
```

Comment:
This should get data from server response? Is this a place holder for response?

Reply:
If someone is trying to get a productId that is invalid, this will be displayed. For example if you put in the url "local/host/4004/description/890" then this will be displayed.

Reply:
Got it. Also, you can provide the content of the response from the server if productID is not found.